nzediegwu1/blog-connect-web
# #161525705 Setup eslint for linting with houndci

#### What does this PR do?
Setup eslint with houndci for automatic code linting

#### Description of Task to be completed?
- install eslint with airbnb
- configure eslintrc for react
- create .eslintignore file and ignore test folder
- create houndci yml and configure settings

#### How should this be manually tested?
- In code editor, make a linting error to observe eslint flag it as error
- When a PR is raised, houndci runs and flags linting errors as comments on the PR

#### Any background context you want to provide?
- ESLint is the required linter for the project. It'll help check for style violations and flag them.
- Houndci automatically points out code style violations so that the team can focus on reviewing actual implementation

#### What are the relevant pivotal tracker stories?
[#161525705](https://www.pivotaltracker.com/story/show/161525705), [#161525743](https://www.pivotaltracker.com/story/show/161525743)

#### Screenshots (If applicable)
N/A
